3_javascript/javascript1/javaScript/script.js

```diff
@@ -0,0 +1,71 @@
+//Calculate age based on year of birth and future year.//
+let yearOfBirth = 1990;
+let yearFuture = 2045;
+let age = (yearFuture - yearOfBirth);
+//log the calculated age to console//
+console.log(age);
+//Dog Age Calculator//
+let dogYearOfBirth = 2017
+let dogYearFuture = 2027
+let dogYear
+let shouldShowResultInDogYears = true; //boolean
+
+if (shouldShowResultInDogYears) {
+dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+
+}
+console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture);
+console.log("Your dog will be " + dogYear + "dog years old in " + dogYearFuture);
+
+//Price House Estimator//
+
+//Peter's house details//
+let houseWidthPeter = 8;
+let houseDepthPeter = 10;
+let houseHeightPeter = 10;
+let gardenSizeInM2Peter = 100;
+let houseCostPeter = 2.500000;
```

Comment:
I would suggest to use here const instead of let, since those values is not going to change, right? 😄